extra-tests/classes/RollupFullRecalcTests.cls

```diff
@@ -937,7 +936,16 @@ private class RollupFullRecalcTests {
 
     acc = [SELECT AnnualRevenue FROM Account];
     System.assertEquals(6, acc.AnnualRevenue);
-    System.assertEquals('Completed', [SELECT Status FROM AsyncApexJob WHERE JobType = 'BatchApexWorker'].Status);
+    System.assertEquals(
+      'Completed',
+      [
+        SELECT Status, ApexClass.Name
```

Comment:
Very small detail, the field `ApexClass.Name` seems like it's unused/not needed in the `SELECT`

Reply:
Ah yeah, I had added that while debugging the test and then forgot to remove it 😅